hack/perf-ibmcloud-openshift.sh

```diff
@@ -314,6 +333,13 @@ install_demo_apps() {
 }
 
 install_workloads() {
+  SCC_SERVICE_ACCOUNTS=""
+  for (( c=0; c<WORKLOAD_NAMESPACES; c++ ))
+  do
+    SCC_SERVICE_ACCOUNTS="${SCC_SERVICE_ACCOUNTS}- \"system:serviceaccount:depth-$c:default\"
+"
```

Comment:
I think you want to put the quote on the line above, no? That seems to do the same thing and won't look weird.
```suggestion
    SCC_SERVICE_ACCOUNTS="${SCC_SERVICE_ACCOUNTS}- \"system:serviceaccount:depth-$c:default\""
```

Reply:
oh, jsut saw this.

Add a comment in here then, to explain. otherwise, people (me) will see it and get a twitch in their eye..it just  looks wrong.